Python_Backend/StateGenerator.py

```diff
@@ -0,0 +1,26 @@
+import pickle
+import pandas as pd
+class StateGenerator:
+    def __init__(self, model_path, model_type):
+        self.model_path = model_path
+        self.model_type = model_type
+        self.states = None
+        if (self.model_type == "FACE"):
+            self.states = ["NEUTRAL", "SMILE", "FROWN"]
+        elif (self.model_type == "IRIS"):
+            self.states = ["EYES_UP", "EYES_DOWN", "EYES_LEFT", "EYES_RIGHT", "EYES_CENTRE"]

```

Comment:
Would an eyes-closed state be helpful here aswell? Also just thinking about what would happen if they winked. Probably an unlikely case, but one eye open and the other closed might be problematic. Tho the ML could probably be trained to treat one eye closed as the "eyes closed" state.

Reply:
I think for the kid we have, these states are sufficient. I think eyes closed is likely to get confused with eyes down.

Reply:
For blinking, if it is recognized as an eyes down then its likely to only be a few times, in that case we just ignore it. I've opened issue #133 to remind us to check for this.